.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "xxHash"]
```

Comment:
You're changing the build instructions here. It's not possible to just do a normal clone of the repo and fire a `make` call anymore.

Looking at the current master, It's general not a good advice having no build instructions at all. But you're introducing a stupid trap door here, which warrants a comment a.k.a. build documentation.

While reading the XXH64 website, it looks pretty promising.